datafusion/common/src/stats.rs

```diff
@@ -25,7 +25,7 @@ use arrow_schema::Schema;
 
 /// Represents a value with a degree of certainty. `Precision` is used to
 /// propagate information the precision of statistical values.
-#[derive(Clone, PartialEq, Eq, Default)]
+#[derive(Clone, PartialEq, Eq, Default, Copy)]
```

Comment:
This change only affects the type when the templated type is `Copy`

So in other words this doesn't make it easy to accidentally copy `Precision<ScalarValue>` only copies when the underlying `T` also supports copy 

I found https://users.rust-lang.org/t/copy-bound-on-type-parameters/58928 helpful

Reply:
BTW: You can always manually implement the `Copy` trait if you want to have fine grained control about the bounds:

```rust
impl<T> Copy for Precision<T> where: TDebug + Clone + PartialEq + Eq + PartialOrd + Clone {}
```

Reply:
👍  -- I think that is basically what `#[derive(Debug)]` does in this case, which is kind of cook